torat_client/rpcapi.go

```diff
@@ -20,7 +21,16 @@ import (
 type API int
 
 func (a *API) Shred(s *shared.Shred, r *shared.Void) error {
-	return s.Conf.Path(s.Path)
+	shredconf := shred.Conf{
+		Times:  s.Times,
+		Zeros:  s.Zeros,
+		Remove: s.Remove,
+	}
+
+	if err := shredconf.Path(s.Path); err != nil {
+		return err
+	}
+	return nil
```

Comment:
This can just be
```
return shredconf.Path()
```

Reply:
Didn't realize shred with just return nil if no issues.

Reply:
Otherwise it wouldn't really make sense to compare an error to nil.